src/Microsoft.Health.Fhir.Shared.Core/VersionSpecificModelInfoProvider.cs

```diff
@@ -64,9 +64,12 @@ public Type GetTypeForFhirType(string resourceType)
             return ModelInfo.GetTypeForFhirType(resourceType);
         }
 
-        public EvaluationContext GetEvaluationContext(ITypedElement element)
+        public EvaluationContext GetEvaluationContext(ITypedElement element, Func<string, ITypedElement> elementResolver = null)
```

Comment:
Why not remove the ITypedElement from here and the interface? It doesn't look like it is being used.